### PR TITLE
parquet test and csv, parquet writes restructure

### PR DESCRIPTION
**Pull request by Sandeepa1995**

### CODE REVIEW

File: python/pycylon/data/csv.pyx [hunk 1]
```diff
@@ -47,4 +48,17 @@ def read_csv(context, path, csv_read_options) -> Table:
         if status.is_ok():
             return pycylon_wrap_table(cn_table)
         else:
-            raise Exception(f"Table couldn't be created from CSV: {status.get_msg().decode()}")
\ No newline at end of file
+            raise Exception(f"Table couldn't be created from CSV: {status.get_msg().decode()}")
+
+def to_csv(table, path, csv_write_options):
+        """
+        creating a csv file with PyCylon table data
```
Comment (supunkamburugamuve) [inline python/pycylon/data/csv.pyx:55]: I'm not sure in Python API, we should associate these with the table?

Comment (supunkamburugamuve) [reply python/pycylon/data/csv.pyx:55]: @vibhatha could you comment

Comment (vibhatha) [reply python/pycylon/data/csv.pyx:55]: @supunkamburugamuve  I am taking a look.

Comment (vibhatha) [reply python/pycylon/data/csv.pyx:55]: as I can remember I made this to_csv as a function in the table. 
for csv.pyx we shouldn't have dependencies with the table class. 
correct me if I am wrong.

Comment (Sandeepa1995) [reply python/pycylon/data/csv.pyx:55]: Yes, to_csv was originally a part of Table. But with WriteCSV being moved out of Table, in the cpp code, I mirrored the same with the Python code (and altered any instances where it was used like in tests). 
Should the Python code be kept as it was? Where to_csv is a part of Table in Python but not in the cpp version?

Comment (supunkamburugamuve) [reply python/pycylon/data/csv.pyx:55]: The reasoning behind was to keep the Python API as user friendly as possible. Lets keep it as previously for now.

Comment (Sandeepa1995) [reply python/pycylon/data/csv.pyx:55]: Will do. I'll make the changes and update the pull request.

Comment (supunkamburugamuve) [reply python/pycylon/data/csv.pyx:55]: +1

**Review by supunkamburugamuve — APPROVED** [review]

**Review by supunkamburugamuve — APPROVED** [review]

**Pull request merged**